go/ql/lib/change-notes/2023-12-08-find-more-callees-for-captured-functions.md

```diff
@@ -0,0 +1,4 @@
+---
+category: fix
+---
+* When a function literal refers to a variable which has function type, there was a bug that meant that we were not able to find any possible callees for calls to that variable. This has now been fixed.
```

Comment:
```suggestion
category: minorAnalysis
---
* `CallNode::getACallee` and related predicates now recognise more callees accessed via a function variable, in particular when the callee is stored into a global variable or is captured by an anonymous function. This may lead to new alerts where data-flow into such a callee is relevant.
```